### ISSUE TITLE
16 blog posts since Feb '18 missing from Category pages

### ISSUE THREAD

**Issue by ToniBonittoGSA** [GSA/digitalgov.gov#791]
I noticed it when I saw that [Access. for Teams post](https://digital.gov/2018/07/10/introducing-accessibility-for-teams/) was missing from the [Accessibility Category page](https://digital.gov/categories/accessibility/) and the [UX Category page](https://digital.gov/categories/ux/), despite having those categories assigned in the code.

All 16 on first page of [All Posts](https://digital.gov/posts/): 

**Health Tech Sprint Aims at Improving Care Access and Experience**
Nov 2, 2018 | Gil Alterovitz, PhD, Justin Koufopoulos, Kristen Honey, PhD
https://digital.gov/2018/11/02/health-tech-sprint-aims-at-improving-care-access-experience/
**Missing from**: CODE

**GSA Steps Up Security for .gov**
Oct 29, 2018 | Cameron Dixon, Marina Fox
https://digital.gov/2018/10/29/gsa-steps-up-security-for-dotgov/ 
**Missing from**: CODE, MANAGING DIGITAL

**A Design Challenge to Improve How School Data is Reported**
Sep 26, 2018 | Office of Educational Technology
https://digital.gov/2018/09/26/office-educational-technology-hold-design-challenge-supporting-data-reporting-requirements/ 
**Missing from**: CHALLENGES, DATA

**Want a More Innovative Agency? Encourage Employees to Experiment**
Sep 19, 2018 | Brian Fox
https://digital.gov/2018/09/19/want-a-more-innovative-agency-encourage-employees-to-experiment/ 
**Missing from**: MANAGING DIGITAL

**Collaborating for Better Design, Technology, and User Experience**
Aug 30, 2018 | John Sullivan
https://digital.gov/2018/08/30/collaborating-for-better-design-technology-user-experience/ 
**Missing from**: ACCESSIBILITY, DESIGN, MANAGING DIGITAL, UX

**Experiments in Tweaking Agile for UX**
Jul 17, 2018 | Meghan Lazier, Jessica Skretch
https://digital.gov/2018/07/17/experiments-in-tweaking-agile-for-ux/ 
**Missing from**: DESIGN, MANAGING DIGITAL, UX

**Introducing Accessibility for Teams**
Jul 10, 2018 | Maya Benari
https://digital.gov/2018/07/10/introducing-accessibility-for-teams/ 
**Missing from**: ACCESSIBILITY, UX

**How One Team is Making the Transition to Agile**
Jun 26, 2018 | Andrew J. Stauffer
https://digital.gov/2018/06/26/how-one-team-making-transition-agile/ 
**Missing from**: MANAGING DIGITAL

**Thinking About Going Agile? 5 Benefits Your Office Will Reap With Agile Methods**
Apr 3, 2018 | Brian Fox
https://digital.gov/2018/04/03/thinking-about-going-agile-5-benefits-your-office-will-reap-with-agile-methods/
**Missing from**: MANAGING DIGITAL

**New USAGov Program Aims to Help Agencies Market Within the Federal Government**
Apr 2, 2018 | Claire Loxsom
https://digital.gov/2018/04/02/marketing-government-through-agency-partnerships/ 
**Missing from**: CONTENT, DESIGN, METRICS, SOCIAL MEDIA, WRITING

**The Value of Federal Government Data**
Mar 14, 2018 | Bill Brantley
https://digital.gov/2018/03/14/data-briefing-value-federal-government-data/ 
**Missing from**: DATA, METRICS

**Thinking About Going Agile? Consider Mass Training to Facilitate the Change**
Mar 13, 2018 | Brian Fox
https://digital.gov/2018/03/13/thinking-about-going-agile-consider-mass-training-facilitate-change/ 
**Missing from**: MANAGING DIGITAL

**Redesigning an Intranet Site: Final Stages, Launch, and Lessons Learned**
Mar 7, 2018 | Karla Blaine
https://digital.gov/2018/03/07/redesigning-an-intranet-site-final-stages-launch-lessons-learned/ 
**Missing from**: ACCESSIBILITY, CONTENT, DATA, DESIGN, MANAGING DIGITAL, UX

**Why Your Team Needs Its Own Style Guide and Where to Start**
Feb 13, 2018 | Meghan Lazier
https://digital.gov/2018/02/13/why-your-team-needs-its-own-style-guide-where-start/ 
**Missing from**: ACCESSIBILITY, CONTENT, DESIGN, MOBILE, UX

**Trends on Tuesday: Because Seeing is Believing—Augmented Reality in the Federal Government**
Feb 6, 2018 | Jada Jones
https://digital.gov/2018/02/06/trends-on-tuesday-because-seeing-believing-augmented-reality-in-federal-government/ 
**Missing from**: MANAGING DIGITAL

**Using Journey Mapping to Streamline Processes Across Agencies**
Feb 5, 2018 | Laura Godfrey
https://digital.gov/2018/02/05/using-journey-mapping-streamline-processes-across-agencies/ 
**Missing from**: CONTENT, DESIGN, METRICS, UX

## it's done when …
- [ ] all 16 posts appear on their correct Category pages

**Comment by jeremyzilar** [comment]
I am looking into this.

**Comment by ToniBonittoGSA** [comment]
@jeremyzilar added 2 more
